deps/v8/src/inspector/PRESUBMIT.py

```diff
@@ -17,7 +17,7 @@ def _CompileScripts(input_api, output_api):
   local_paths = [f.LocalPath() for f in input_api.AffectedFiles()]
 
   compilation_related_files = [
-    "js_protocol.json"
+    "js_protocol.json",
```

Comment:
This is V8 issue. Can you report it there?

Reply:
It looks like this file was deleted upstream in [October last year](https://github.com/v8/v8/commit/ebd070ec2f94a6834190606b5d31cdf8f193d38a#diff-3ed9b21ff0ad2ecba12f586efa3f0089). I'll back this change out in any case.